packages/lib/src/command-util.ts

```diff
@@ -104,15 +104,25 @@ export async function stringGetIdFromUser<L>(fieldInfo: Sorting, list: L[], prom
 	return inputId
 }
 
+/**
+ * Note that not all functions that use this interface support all options. Check the
+ * `chose<Thing>` method itself.
```

Comment:
`chose<Thing>` makes it seem like a method named `chose` which returns a `Thing`.